NEWS.md

```diff
@@ -1,3 +1,7 @@
+# hubValidations 0.3.0
+
+* Introduce an `output_type_id_datatype` argument to `validate_pr()`, `validate_submission()`, `validate_model_data()` and `expand_model_out_grid()` and set default value to `"from_config"`. This default means the data type specified in the `output_type_id_datatype` property in `tasks.json` (introduced in schema version `v3.0.1`) is used to cast the hub level `output_type_id` column data type. If not set in the config, the functions fall back to `"auto"` which detects the simplest data type that can represent all output type id values across all output types and rounds. The argument also allows hub administrators to override this setting manually during validation.
```

Comment:
I think it's ok honestly because there's a lot more detail in the actual function documentation and I don't think adding _automatically_ there adds that much more clarity. Feels more like repetition.